moonshot/reimage_watch.exp

```diff
@@ -0,0 +1,155 @@
+expect "Install Ubuntu"
+send -- "e"
+set timeout 60
+expect "Press Ctrl-x to start, Ctrl-c for a command prompt or Escape to"
+expect "initrd.gz                    \[10;01H                                                                               \[11;01H                                                                               \[12;01H                                                                               \[13;01H                                                                               \[14;01H                                                                               \[15;01H                                                                               \[16;01H                                                                               \[17;01H                                                                               \[18;01H                                                                               \[04;80H \[18;80H \[04;01H"
+#expect -exact "\[0m\[30m\[40m\[2J\[01;01H\[0m\[37m\[40m\[0m\[30m\[40m\[2J\[01;01H\[0m\[37m\[40m\[20;07HPress Ctrl-x to start, Ctrl-c for a command prompt or Escape to     
+#\r      discard edits and return to the menu. Pressing Tab lists possible   
+#\r      completions.                                                        \[04;01Hsetparams 'Install Ubuntu 16.04.0 LTS x86_64 on moonshot'                      \[05;01H                                                                               \[06;01H    linuxefi images/Ubuntu-16.04.0-x86_64-server/linux ro auto=true url=http:/\\
+#\r\[07;01H/repos/repos/kickstart/ubuntu_16.04_x64_moonshot.preseed priority=critical int\\
+#\r\[08;01Herface=auto                                                                    \[09;01H    initrdefi images/Ubuntu-16.04.0-x86_64-server/initrd.gz                    \[10;01H                                                                               \[11;01H                                                                               \[12;01H                                                                               \[13;01H                                                                               \[14;01H                                                                               \[15;01H                                                                               \[16;01H                                                                               \[17;01H                                                                               \[18;01H                                                                               \[04;80H \[18;80H \[04;01H"
+sleep 1
+send -- "\[B"
+expect -exact "\[05;01H"
+send -- "\[B"
+expect -exact "\[06;01H"
+send -- "\[B"
+expect -exact "\[09;01H"
+send -- "\[D"
+expect -exact "\[08;12H"
+send -- " "
+sleep 1
+send -- "PUPPET_PASS="
+sleep 4
+send -- "$kickstart"
+sleep 5
+send -- ""
+sleep 60
```

Comment:
The only real difference between this and the reimage.exp is this wait to "watch" the install start.